tools/release

```diff
@@ -51,17 +51,19 @@ if 'upload' not in sys.argv:
     sys.exit(0)
 
 # Register with the Python Package Index (PyPI)
-print( 'Registering with PyPI...')
+print( 'Not Registering with PyPI, registering with setup.py is insecure as communication is not encrypted')
 cd(ipdir)
-sh('./setup.py register')
+# sh('./setup.py register')
 
 # Upload all files
-sh(sdists + ' upload')
+sh(sdists)
 for py in ('2.7', '3.4'):
-    sh('python%s setupegg.py bdist_wheel upload' % py)
+    sh('python%s setupegg.py bdist_wheel' % py)
+
+print('Will not upload with setuptools as upload connection is insecure. Please use `twine upload dist/*` to upload the files to PyPI')
```

Comment:
Sure in the long run, but we have a people visiting today in 1/2 an hour, and I was going to make that in a separate PR when I have time unless So does it in the meantime. (I guess you can use twine as a library). 

Also I'm sure there will be much bikeshed so separate seem better to iterate quickly.